portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsSavingAnalysis.testcase

```diff
@@ -711,18 +711,7 @@ definition {
 	test CanPaginationAnalysisReportList {
 		var eventAnalysisNameList1 = "Save Analysis 0,Save Analysis 1,Save Analysis 2,Save Analysis 3";
 		var eventAnalysisNameList2 = "Save Analysis 4,Save Analysis 5,Save Analysis 6,Save Analysis 7";
-
-		task ("Fill fields and create custom event") {
-			var customEventName = ACCustomEvents.generateCustomEventName();
-
-			ACCustomEvents.createCustomEvent(
-				customEventName = ${customEventName},
-				timeToSendCustomEvent = 1);
-		}
-
-		task ("Close sessions") {
-			ACUtils.closeAllSessionsAndWait();
-		}
```

Comment:
Hey Vini, can you explain to me why you are deleting these lines of code (same thing with the other lines)? Isn't this part of the test?

Reply:
Hey @manuferreira, The purpose of this test is to check if the pagination of the event analysis list is working or not. So it is not necessary to create a custom event to check this, so I removed this part and put the test to create the event analysis using the default events.

I also removed an assert that it made to check if the event analysis had data before creating it. For this test case, it is not important if we have data in the event analysis or not.

I believe that this way we can make the test more simple and targeted to the objective and avoid problems that could happen by generating the custom event (sometimes the emulator dies when we generate the custom event)

What do you think?

Reply:
sounds good to me!